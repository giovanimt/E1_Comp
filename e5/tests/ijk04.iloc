loadI 14 => r0 
addI rfp, 4 => r1 
store r0 => r1 
loadI 16 => r2 
addI rfp, 8 => r3 
store r2 => r3 
addI rfp, 8 => r6 
load r6 => r7 
addI rfp, 4 => r4 
load r4 => r5 
add r5, r7 => r8 
addI rfp, 0 => r9 
store r8 => r9 
addI rfp, 8 => r12 
load r12 => r13 
addI rfp, 0 => r10 
load r10 => r11 
add r11, r13 => r14 
addI rfp, 4 => r15 
store r14 => r15 
addI rfp, 4 => r18 
load r18 => r19 
addI rfp, 0 => r16 
load r16 => r17 
add r17, r19 => r20 
addI rfp, 8 => r21 
store r20 => r21 
