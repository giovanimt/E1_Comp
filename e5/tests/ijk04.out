00035429 30
00035433 46
00035437 76
